api/Document.json

```diff
@@ -322,32 +376,68 @@
             "ie": {
               "version_added": "4"
             },
-            "opera": {
-              "version_added": "≤12.1"
-            },
-            "opera_android": {
-              "version_added": "≤12.1"
-            },
+            "opera": [
+              {
+                "version_added": "51",
+                "notes": "Starting in Opera 52, this property is readonly."
+              },
+              {
+                "version_added": "15",
+                "version_removed": "51",
+                "partial_implementation": true,
+                "notes": "Only supported for <a href='https://developer.mozilla.org/docs/Web/API/HTMLDocument'><code>HTMLDocument</code></a>, not all <code>Document</code> objects."
+              },
+              {
+                "version_added": "≤12.1",
```

Comment:
This and more Opera ranges that need to be merged into the `partial_implementation` range remain. Or are there some of these things that were actually on both `HTMLDocument` and `XMLDocument` in Presto?

Reply:
Ranges are now merged together!